### PR TITLE
Rework testnet and Delete working groups

### PR DESCRIPTION
**Pull request by thomasheremans**
- Delete working groups outdated page
- Update testnet page

### CODE REVIEW

File: docs/testnet.md [hunk 1]
```diff
@@ -15,14 +13,11 @@ The following is a break down of the intended testnet roadmap, the features that
 - [x] [Dev Net 1](#devnet-1)
 - [x] [Dev Net 2](#devnet-2)
 - [x] [Athena Public Testnet 1](#athena-public-testnet-1)
-- [ ] [Bia Attack net](#bia-attack-net)
-- [ ] [Circe Public Testnet 2](#cerce-public-testnet-ii)
-- [ ] [Demeter Red/Blue net](#demeter-redblue-net)
+- [ ] [Bia Public Testnet 2](#bia-public-testnet-2)
+- [ ] [Circe Attack Net](#cerce-attack-net)
```
Comment (OisinKyne) [inline docs/testnet.md:17]: Looks like I've been mispelling it and it's Circe not Cerce 😬  https://en.wikipedia.org/wiki/Circe#:~:text=Circe%20(%2F%CB%88s%C9%9C%CB%90rs,knowledge%20of%20potions%20and%20herbs.

File: docs/testnet.md [hunk 2]
```diff
@@ -70,116 +48,68 @@ A core focus of this devnet was to collect network performance data. This was th
...
-
-## Demeter Red/Blue Net
-
-The final planned testnet before a prospective look at mainnet deployment is a testnet that takes inspiration for the Cyber Security industry and makes use of Red Teams and Blue Teams.
-
-In Cyber Security, Red team are on offense, Blue team are on defence. In Obol's case, Operators will be grouped into clusters based on application and are assigned to either red team or blue team in secret. Once the validators are active, it will be the red teamers goal to disrupt the cluster to the best of their ability, and their rewards will be based on how much worse the cluster performs than optimal.
+- Engage the wider Solo and Professional Ethereum Staking Community.
+- Get integration feedback.
+- Build confidence in Charon after running DVs on an Ethereum testnet.
+- Prove Charon superior performance and benefits compared to other DVT solutions.
```
Comment (OisinKyne) [inline docs/testnet.md:93]: Maybe lets cut this point, its probably better left unsaid

File: docs/testnet.md [hunk 3]
```diff
@@ -70,116 +48,68 @@ A core focus of this devnet was to collect network performance data. This was th
...
-**Network:** Merged Gnosis Chain
+**Duration:** 2-4 weeks operation, depending on how resilient the clients are
 
 **Goals:**
 
-- Even with incentivised byzantine actors, distributed validators can reliably stay online
-- Charon nodes cannot be DoS'd
-- Demonstrate that fault tolerant validation is real, safe and cost competetive.
-- Charon is feature complete and ready for audit
+- Break charon, launchpad, API in multiple ways after having tested the happy path during the previous testnets
```
Comment (OisinKyne) [inline docs/testnet.md:112]:
```suggestion
- Break charon, the launchpad, and the Obol API in multiple ways after having focused primarily on the happy path during the previous testnets.
```

File: docs/testnet.md [hunk 4]
```diff
@@ -70,116 +48,68 @@ A core focus of this devnet was to collect network performance data. This was th
...
+- Create a number of testing tools to facilitate the disruption of charon, including releasing a p2p network abuser, a fuzz testing client, k6 scripts for load testing/hammering an RPC endpoints and more.
```
Comment (OisinKyne) [inline docs/testnet.md:113]:
```suggestion
- Create a number of testing tools to facilitate the disruption of charon, including releasing a p2p network abuser, a fuzz testing client, k6 scripts for load testing/hammering RPC endpoints and more.
```

**Review by OisinKyne — APPROVED** [review]
Some spelling to fix, remember to fix it in the hyperlink as well as the heading. 

Kind of liked the couple of sentences of story at the start of each testnet as to the 'why' we are doing it. Not the end of the world if we take them out but I feel the page loses a bit of character.

**Comment by thomasheremans** [comment]
Corrected spelling with Grammarly and added back shortened description as otherwise redundant with the goals

**Pull request merged**